products/panos/docs/intro.md

```diff
@@ -16,24 +16,30 @@ keywords:
 ---
 
 # PAN-OS APIs and SDKs
+
 PAN-OS APIs and SDKs allow you to manage next-generation firewalls, directly or via Panorama, using third-party services, applications, or scripts.
 
 ## PAN-OS XML API
+
 The [PAN-OS XML API](./xmlapi.md) is powerful and low-level, allowing you to take full control of every aspect of your security, and build deep integrations with a variety of other systems. You can make XML API calls directly to the firewall, directly to Panorama, or to a firewall via Panorama.
 
 ## PAN-OS REST API
+
 The [PAN-OS REST API](./restapi.md) simplifies access to resources as high-level URIs. You can use this API to create, change, and delete resources.
 
-## ```pan-os-python``` (Python)
-The [```pan-os-python``` SDK framework](./pan-os-python.md) helps interact with PAN-OS devices when your chosen language is Python. It is object-oriented and mimics the traditional interaction with the device via the GUI, CLI or XML API. It is also part of the underpinnings of the [PAN-OS Ansible collection](../../ansible/panos/docs/).
+## `pan-os-python` (Python)
+
+The [`pan-os-python` SDK framework](./pan-os-python.md) helps interact with PAN-OS devices when your chosen language is Python. It is object-oriented and mimics the traditional interaction with the device via the GUI, CLI or XML API. It is also part of the underpinnings of the [PAN-OS Ansible collection](/ansible/docs/panos/).
 
 <!---
 ## ```pan-python``` (Python)
```

Comment:
Do we want to un-triple-tick this line too, for future?

Reply:
That's me, sorry. Learned a bad pattern and replicated it everywhere until I realised it wasn't the way to do it. I mean, it works, but the code sucks. Happy to fix them all, maybe I'll wait until the other changes are merged in though? Like the markdown partials PR, my mid-Nov PR, etc? Then I'll blast away all the triple-ticks? Let me know what you think...